db/migrate/20241203191058_auto_gen_uuid_ecf_id_for_applications.rb

```diff
@@ -0,0 +1,6 @@
+class AutoGenUuidEcfIdForApplications < ActiveRecord::Migration[7.1]
+  def change
+    change_column_default :applications, :ecf_id, from: nil, to: "gen_random_uuid()"
```

Comment:
worth checking on all envs that this value is present incase :) same with users

Reply:
Updated migration to generate ecf_id if nil for User and Application